### ISSUE TITLE
Laravel 5.1

### ISSUE THREAD

**Issue by turzaizsolt** [rmariuzzo/Laravel-JS-Localization#16]
Does this package support laravel 5.1? If so, can you please update the composer.json.

Thnaks.

**Comment by vluzrmos** [comment]
I think... @rmariuzzo is missing... 
My PR was never commented #15

**Comment by rmariuzzo** [comment]
I have been a little busy these days. I will confirm you in a few minutes.
I'm sorry 🙏

**Comment by rmariuzzo** [comment]
Because the directory structure in Laravel 5.1 is not different than Laravel 5. I assume this package will work just fine. However, I will upgrade a Laravel 5 project to 5.1 and see how it looks.

Thanks @turzaizsolt and @vluzrmos

**Comment by vluzrmos** [comment]
Yes @rmariuzzo , the structure doesn't change, I belive you just need to change the composer.json 
```
"php": ">=5.5.9",
"illuminate/support": "5.0.*|5.1.*",
"illuminate/console": "5.0.*|5.1.*"
"illuminate/filesystem": "5.0.*|5.1.*",
```

**Comment by rmariuzzo** [comment]
That could be a PR from you ;) Never mind, I'm testing right now.

**Comment by vluzrmos** [comment]
Thanks! ;)

**Comment by rmariuzzo** [comment]
Thank you too, for pinging me.

Rubens Mariuzzo
rubens@mariuzzo.com

**Comment by rmariuzzo** [comment]
@turzaizsolt and @vluzrmos I have just released the version 1.2.0 which support Laravel 5.1.x and have working tests.

**Comment by turzaizsolt** [comment]
Thanks!
